<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Statistics</class>
 <widget class="QWidget" name="Statistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>187</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Статистика</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QComboBox" name="moduleName">
       <property name="currentText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>С</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateFrom"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>по</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateTill">
           <property name="date">
            <date>
             <year>2020</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="totalMonitoringTime">
       <property name="text">
        <string>Общее время мониторинга: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="totalValue">
       <property name="minimumSize">
        <size>
         <width>257</width>
         <height>52</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="timeInOkState">
           <property name="text">
            <string>Время в состоянии OK:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="timeInFailState">
           <property name="text">
            <string>Время в состоянии FAIL:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="timeInWrongState">
           <property name="text">
            <string>Время в состоянии &quot;Wrong&quot;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="timeOk">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="timeFail">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="timeUnkn">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QProgressBar" name="timeInOkStatePb">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="timeInFailStatePb">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="timeInWrongStatePb">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
